vehicle/autoware_external_cmd_converter/src/node.cpp

```diff
@@ -195,6 +198,10 @@ void ExternalCmdConverterNode::check_topic_status(
 
 bool ExternalCmdConverterNode::check_emergency_stop_topic_timeout()
 {
+  if (current_gate_mode_ && current_gate_mode_->data == tier4_control_msgs::msg::GateMode::AUTO) {
```

Comment:
@shtokuda 
Do we need `current_gate_mode_` empty validation before this?

Reply:
Yes, I believe there is no guarantee that current_gate_mode is not empty because this function is called within a timer callback.

Reply:
```suggestion
  if (!current_gate_mode_) {
    return true;
  }  

  if (current_gate_mode_->data == tier4_control_msgs::msg::GateMode::AUTO) {
```